applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -422,11 +339,8 @@ static void stream_released_cb(struct bt_audio_stream *stream)
 	int ret;
 	LOG_DBG("Audio Stream %p released", (void *)stream);
 
-	if (headsets[AUDIO_CH_L].sink_stream->ep->status.state != BT_AUDIO_EP_STATE_STREAMING
-#if !CONFIG_STREAM_BIDIRECTIONAL
-	    && headsets[AUDIO_CH_R].sink_stream->ep->status.state != BT_AUDIO_EP_STATE_STREAMING
-#endif
-	) {
+	if (headsets[AUDIO_CH_L].sink_stream.ep->status.state != BT_AUDIO_EP_STATE_STREAMING &&
```

Comment:
Is it OK to remove the bidir flag here?

Reply:
It should be, because now we can have bidir using 2 headsets. I also improved this by creating the "ep_state_check()" function